lode-ui/package.json

```diff
@@ -1,12 +1,18 @@
 {
   "name": "lode-ui",
   "version": "0.1.0",
+  "license": "MIT",
   "dependencies": {
+    "@chakra-ui/react": "^1.6.0",
+    "@emotion/react": "^11",
+    "@emotion/styled": "^11",
     "@react-three/drei": "^4.3.2",
     "@react-three/fiber": "^6.0.19",
     "@testing-library/jest-dom": "^5.11.4",
     "@testing-library/react": "^11.1.0",
     "@testing-library/user-event": "^12.1.10",
+    "framer-motion": "^4",
```

Comment:
I'd prefer having the same version ranges for @emotion and framer-motion as for all other dependencies - even if it's written like this in the Getting Started guide of Chakra 😄